src/lib/components/AssistantSettings.svelte

```diff
@@ -125,6 +135,12 @@
 					<span class="text-xs text-gray-500 hover:underline">Click to upload</span>
 				{/if}
 				<p class="text-xs text-red-500">{getError("avatar", form)}</p>
+				{#if (!files || !files[0]) && $page.data.avatarGeneration && !assistant?.avatar}
+					<label class="text-xs text-gray-500">
+						<input type="checkbox" name="generateAvatar" class="text-xs text-gray-500" />
+						Generate avatar using a text-to-image model.
```

Comment:
```suggestion
						Generate avatar
```

maybe just this?

Reply:
maybe `Generate avatar from description` to make it clear it's text to image and where the definition is coming from ? Though not sure about this, lmk what you think

Reply:
done!